imagemagick.yaml

```
@@ -0,0 +1,24 @@
+# from https://github.com/scx/xaralx-flatpak/blob/master/imagemagick.yaml
+name: imagemagick
+config-opts:
+  - --enable-shared
+  - --disable-static
+  - --with-modules
+  - --with-x
+  - --with-threads
+  - --with-magick_plus_plus
+  - --with-gslib
+  - --with-wmf
+  - --with-webp
+  - --with-openexr
+  - --with-rsvg
+  - --with-xml
+  - --with-jbig
+  - --with-openjp2
+sources:
+  - type: archive
+    url: https://github.com/ImageMagick/ImageMagick6/archive/6.9.10-23.tar.gz
+    sha256: 2a0b11952c994b06562dace16a53c01bdd0f715de59483145eb3053902a42361
+# We need it only for the build stage
+cleanup:
+  - *
```

```suggestion
  - '*'
```

this is while the build fail.